conf/capsule.yaml.template

```diff
@@ -1,7 +1,4 @@
 CAPSULE:
-  # Capsule name to be used in the CapsuleVirtualMachine class (will be deprecated)
-  SATELLITE_VERSION_TAG: "@jinja {{this.robottelo.satellite_version | replace('.', '')}}"
-  INSTANCE_NAME: "@format qe-sat{this[capsule].satellite_version_tag}-rhel7-tierX-capsule"
```

Comment:
IS this not required anymore ?

Reply:
I think it's not. I haven't found any references from within robottelo to use this settings.